src/plugin.js

```diff
@@ -138,6 +138,18 @@ class PlaylistMenuItem extends Component {
     const li = document.createElement('li');
     const item = this.options_.item;
 
+    if (item.data) {
```

Comment:
For backwards compatibility we probably want to check if this is an object, and if it isn't don't do anything.